airbyte-api/src/main/java/io/airbyte/api/client/AirbyteApiClient.java

```diff
@@ -46,6 +47,8 @@ public class AirbyteApiClient {
   private final HealthApi healthApi;
   private final DbMigrationApi dbMigrationApi;
 
+  private final InternalApi internalApi;
```

Comment:
I think we can find a better name here. InternalApi describes the conditions in which these endpoints are used, not what they are. All of our other endpoints are labeled / grouped by resources / what they are. Ultimately this API is not internal. It's as public as the everything else, so we should just name it as whatever it is.

Reply:
@xiaohansong forgot to tag you on this one too, since it's related. I think what Charles says here makes sense - we talked about eventually defining an internal API that is deployed and authenticated with separately from the existing API, but until we actually make that separation, I think following the existing convention makes sense. @davinchia does that align with how you're thinking about our APIs evolving over time?

Reply:
Agreed, I'm planning to use /attempt root path.